doc/whats_new.rst

```diff
@@ -353,6 +363,11 @@ API changes summary
      (`#6697 <https://github.com/scikit-learn/scikit-learn/pull/6697>`_) by
      `Raghav R V`_.
 
+   - The parameters ``n_iter`` or ``n_folds`` in old CV splitters are replaced
+     by the new parameter ``n_splits`` since it can provide a consistent 
+     and unambiguous interface to represent the number of train-test splits.
+     By `YenChen Lin`_.
```

Comment:
Add a link to the PR. It kind of gives a back reference from whatsnew to the PR / Issue...